shared/config/superblocks.ts

```diff
@@ -155,12 +155,10 @@ export const notAuditedSuperBlocks: NotAuditedSuperBlocks = {
     SuperBlocks.PythonForEverybody
   ],
   [Languages.Ukrainian]: [
-    SuperBlocks.CodingInterviewPrep,
     SuperBlocks.JsAlgoDataStructNew,
     SuperBlocks.UpcomingPython,
     SuperBlocks.A2English,
     SuperBlocks.B1English,
```

Comment:
```suggestion
    SuperBlocks.B1English
```

If the last line is being removed, the new last line needs to be without comma

Reply:
Thanks Ilenia, I'll keep that in mind :)